docs/README.md

```diff
@@ -1,10 +1,21 @@
+### Note for external contributors
+
+At the moment we are working on migrating the docs to a new static website. Until that is finished the
+`grafana/website` repo is unfortunately private and so docs site cannot be run without access.
+
+We still encourage PRs to make the docs better. Markdown content is still editable in this repo and we
+will make sure the changed content works well on the current docs site. In case you need to supply images
+you can include that in the PR and we will move them to the `grafana/website` repo.
```

Comment:
The Markdown content however, is still public in this repository. We still encourage pull requests to make the docs better, and we will make sure the changed content works well on the current docs site. Include any images in your pull request, and we will move them to the `grafana/website` repository.